packages/react/src/components/message/Alert/index.tsx

```diff
@@ -25,41 +43,66 @@ const icons = {
   error: <CircledCrossMedium size={20} />,
 };
 
+const getColors = ({ theme, type }: { theme: DefaultTheme; type?: AlertType }) => {
+  switch (type) {
+    case "warning":
+      return {
+        background: theme.colors.warning.c30,
+        color: theme.colors.warning.c100,
+      };
+    case "error":
+      return {
+        background: theme.colors.error.c30,
+        color: theme.colors.error.c100,
+      };
+    case "info":
+    default:
+      return {
+        background: theme.colors.primary.c20,
+        color: theme.colors.primary.c90,
+      };
+  }
+};
+
 const StyledAlertContainer = styled.div<{ type?: AlertType }>`
   ${(p) => {
-    switch (p.type) {
-      case "warning":
-        return css`
-          background: ${p.theme.colors.warning.c30};
-          color: ${p.theme.colors.warning.c100};
-        `;
-      case "error":
-        return css`
-          background: ${p.theme.colors.error.c30};
-          color: ${p.theme.colors.error.c100};
-        `;
-      case "info":
-      default:
-        return css`
-          background: ${p.theme.colors.primary.c20};
-          color: ${p.theme.colors.primary.c90};
-        `;
-    }
+    const { background, color } = getColors({ theme: p.theme, type: p.type });
+    return css`
+      background: ${background};
+      color: ${color};
+    `;
   }}
+  overflow-wrap: break-all;
```

Comment:
ah yeah indeed I was trying both and ended up leaving this weird mix haha, and actually it's useless because the text already wraps properly without this property